cmd/grr/workflow.go

```diff
@@ -104,3 +106,35 @@ func watchCmd() *cli.Command {
 	}
 	return cmd
 }
+
+func exportCmd() *cli.Command {
+	cmd := &cli.Command{
+		Use:   "export <jsonnet-file> [-d <dashboard-dir>]",
+		Short: "render Jsonnet and save to a directory",
+	}
+	targets := cmd.Flags().StringSliceP("target", "t", nil, "dashboards to target")
+	dashboardDir := cmd.Flags().StringSliceP("destination", "d", nil, "destination folder")
```

Comment:
```suggestion
	dashboardDir := cmd.Flags().StringP("destination", "d", "", "destination folder")
```

Seems like we should be anticipating a string instead of []string.

Reply:
Good catch. But removed anyway!